README.rst

```diff
@@ -111,6 +111,18 @@ or
 
     python3 manage.py populate-db --all
 
+To create a superuser for local testing, in gamification.env set
+
+.. code-block:: bash
+
+    EMAIL_USE_TLS=false
```

Comment:
When this was set to true and I attempted to create a superuser I received this error: `KeyError: 'EMAIL_USE_TLS'
`